src/app/core/pipes/date-format.pipe.ts

```diff
@@ -0,0 +1,10 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+import * as moment from 'moment';
+
+@Pipe({ name: 'appDateTimeToDate' })
+export class DateFormatPipe implements PipeTransform {
+    transform(dateTime: moment.Moment) {
+        return moment(dateTime).format();
```

Comment:
I've created a simmilar pipe for my pr. Let's extend it and make it parametrized:

    transform(dateTime: moment.Moment, timeFormat: string) {
        return timeFormat !== undefined ? dateTime.format(timeFormat) : dateTime.format();
    }

you can then call it like so: {{date | momentFormat:'dd/mm/yyy'}}

Reply:
And call it generically: MomentFormatPipe